base/rootfs/etc/cont-init.d/00-load-secrets.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/with-contenv bash
+set -e
+
+# Read any secret files specified in environment variables.
+echo "$(env | grep '=secret:')" | while read line
```

Comment:
Could you put instructions into the README? It looks like you have to do a specific thing with a filename to make this work, and we shouldn't assume people will go read the code to figure it out.

Reply:
And can you add your SECRETfile as a template to this repo?

Reply:
If you scroll to "By .env technique" here you can see how to load a second .yml file by changing your COMPOSE_FILE declaration in your .env file https://stackoverflow.com/questions/55650342/import-docker-compose-file-in-another-compose-file

Reply:
I put example secret files in https://github.com/Islandora-Devops/isle-dc/pull/123